cmd/backup-manager/app/util/util.go

```diff
@@ -135,6 +135,37 @@ func ConstructBRGlobalOptionsForBackup(backup *v1alpha1.Backup) ([]string, strin
 	return args, remotePath, nil
 }
 
+// ConstructMydumperOptionsForBackup constructs mydumper options for backup
+func ConstructMydumperOptionsForBackup(backup *v1alpha1.Backup) []string {
+	var args []string
+	defaultOptions := []string{
+		"--long-query-guard=3600",
+		"--tidb-force-priority=LOW_PRIORITY",
+		"--verbose=3",
+		"--compress-protocol",
+		"--threads=16",
+		"--rows=10000",
+		"--skip-tz-utc",
+	}
+	defaultTableRegexOptions := []string{
+		"--regex",
+		"^(?!(mysql|test|INFORMATION_SCHEMA|PERFORMANCE_SCHEMA|METRICS_SCHEMA|INSPECTION_SCHEMA))",
+	}
+	config := backup.Spec.Mydumper
+	if config == nil {
+		args = append(args, defaultOptions...)
+		args = append(args, defaultTableRegexOptions...)
+		return args
+	}
+	if len(config.Options) != 0 {
+		args = append(args, config.Options...)
```

Comment:
What if users want to overwrite some of the default options?

Reply:
The default options are missed here.

Reply:
If the user sets the options of mydumper,  we will use the user's own settings directly. Since the user has started to customize the options, I think there is no need to add some other options that the user does not know.